jobs/go/http/dapr.yaml

```diff
@@ -1,13 +1,13 @@
 version: 1
 apps:
-  - appDirPath: ./job-service/
-    appID: job-service
+  - appID: job-service-http
+    appDirPath: ./job-service/
     appPort: 6200
     daprHTTPPort: 6280
     schedulerHostAddress: localhost
```

Comment:
While were making these changes can you remove this line within all the dapr yaml files @marcduiker ?

Reply:
If you have the following versions locally this should work fine.
CLI version: 1.15.0-rc.3 (https://github.com/dapr/cli/pull/1479)
Runtime version: 1.15.0-rc.8 (https://github.com/dapr/dapr/pull/8403)